ArticleTemplates/assets/js/modules/messenger.js

```diff
@@ -0,0 +1,198 @@
+/* global Promise */
```

Comment:
I've just checked and only 11% of our Android userbase are running < 4.4.4. Perhaps we can make the call that as this only likely get smaller we can take this hit.